lib/tether/tether.go

```diff
@@ -251,20 +252,32 @@ func (t *tether) setNetworks() error {
 }
 
 func (t *tether) setMounts() error {
+	// provides a lookup from path to volume reference.
+	NameLookupMap := make(map[string]string, 0)
```

Comment:
yeah and it stuters the type :/ I have been thinking about what to call this one.